daemon/src/send_async_safe.rs

```diff
@@ -0,0 +1,28 @@
+use async_trait::async_trait;
+
+#[async_trait]
+pub trait SendAsyncSafe<M>
+where
+    M: xtra::Message<Result = ()>,
+{
+    /// Send a message to an actor without waiting for them to handle
+    /// it.
+    ///
+    /// As soon as this method returns, we know if the receiving actor
+    /// is alive. If they are, the message we are sending will
+    /// eventually be handled by them, but we don't wait for them to
+    /// do so.
+    async fn send_async_safe(&self, msg: M) -> Result<(), xtra::Disconnected>;
+}
+
+#[async_trait]
+impl<A, M> SendAsyncSafe<M> for xtra::Address<A>
+where
+    A: xtra::Handler<M>,
+    M: xtra::Message<Result = ()>,
+{
+    async fn send_async_safe(&self, msg: M) -> Result<(), xtra::Disconnected> {
+        #[allow(clippy::disallowed_method)]
+        self.do_send_async(msg).await
```

Comment:
Do a `tokio::spawn` here and use `.send` and log the error if it happens.

Reply:
Sorry, for this one there is no error but we can do that for the TODO above!

Reply:
I don't understand these two comments, could you rephrase?

Reply:
Ignore this, I'll post again!